src/game_api/rpc.cpp

```diff
@@ -60,17 +57,17 @@ float get_zoom_level()
     return State::get().get_zoom_level();
 }
 
-void list_items()
-{
-    auto state = State::get();
-    auto player = state.items()->player(0);
-    if (player == nullptr)
-        return;
-    for (auto& item : state.layer(player->layer)->items())
-    {
-        DEBUG("Item {} {:x}, {}", item->uid, item->type->search_flags, item->position_self());
-    }
-}
+//void list_items()

```

Comment:
You wanna remove this code if it's not used?

Reply:
oh, i forgot, i commented that to see if it will fail to build

i guess is not needed? seams like pure debug leftover stuff